app/policies/webhook_policy.rb

```diff
@@ -56,4 +56,12 @@ def resolve
       end
     end
   end
+
+  protected
+
+  def create_or_manage?(perm)
```

Comment:
Maybe just `can?`? It's a local thing, so I guess it's fine. What do you think?

Reply:
I thought about it, but then I realized that it might be confusing, because it's not valid for `show`, for example. So, I'd rename it only if we add more stuff into it with the same implementation.

Reply:
It makes sense! :+1: